docs/log-rotation.md

```diff
@@ -0,0 +1,102 @@
+# Log rotation in Percona Server for MongoDB
+
+When you start Percona Server for MongoDB with a `--logpath` or `systemLog.path` setting, it writes logs to a file instead of standard output. These logs include detailed activity and operation information. 
+
+Percona Server for MongoDB has server and [audit logs](audit-logging.md). To maintain system performance, manage disk space, and meet audit and compliance requirements, you must effectively manage these logs. 
+
+Percona Server for MongoDB provides multiple log rotation mechanisms to help you achieve this goal. You can:
+
+- Send a SIGUSR1 signal to the `mongod` or `mongos` process.
+
+- Run the `logRotate` command on `mongosh`. Note that you must connect to each replica set member and start log rotation for it.
+
+- Use the Linux/Unix `logrotate` utility with proper configuration.
+
+## Log rotation considerations
+
+Percona Server for MongoDB does not rotate logs automatically. You must trigger log rotation manually or integrate with external tools like the Linux/Unix `logrotate` utility. Log rotation is not propagated across replica set members; you must rotate logs on each node independently.
+
+## Rotation behavior configuration
+
+Percona Server for MongoDB supports two log rotation strategies:
+
+* `rename` (default): Renames the current log file by appending a UTC timestamp to the file name. This log file is closed. Percona Server for MongoDB creates a new log file and writes all new log entries to this file.
+* `reopen`: Closes and reopens the log file, expecting an external process (e.g., `logrotate`) to rename the file beforehand.
```

Comment:
should be used with https://www.mongodb.com/docs/manual/reference/configuration-options/#mongodb-setting-systemLog.logAppend set to true